satpy/readers/olci_nc.py

```diff
@@ -177,21 +165,21 @@ def __init__(self, filename, filename_info, filetype_info, cal,
                  engine=None):
         """Init the file handler."""
         super(NCOLCI1B, self).__init__(filename, filename_info,
-                                       filetype_info)
+                                       filetype_info, engine)
         self.cal = cal.nc
 
-    @staticmethod
-    def _get_items(idx, solar_flux):
-        """Get items."""
-        return solar_flux[idx]
-
     def _get_solar_flux(self, band):
         """Get the solar flux for the band."""
         solar_flux = self.cal['solar_flux'].isel(bands=band).values
         d_index = self.cal['detector_index'].fillna(0).astype(int)
 
-        return da.map_blocks(self._get_items, d_index.data,
-                             solar_flux=solar_flux, dtype=solar_flux.dtype)
+        return da.map_blocks(self._take_indices, d_index.data,
+                             data=solar_flux, dtype=solar_flux.dtype)
+
+    @staticmethod
+    def _take_indices(idx, data):
```

Comment:
Just putting this here to sanity check, dask distributed can handle static methods right?

Reply:
no clue. I'll make it a function just in case.